.rocket.yml

```diff
@@ -0,0 +1,31 @@
+---
```

Comment:
PackageConfig please! (sorry)

Reply:
Np i thought was too complex for it :) will move it 👍

Reply:
Dog fooding mainly, also open to helping figure out how to reduce that complexity either in PackageConfig, or Rocket

Reply:
For example, auto running `scripts/deploy/before.sh`,  `scripts/deploy/after.sh` if it exists etc 

We could steal some ideas form https://www.npmjs.com/package/release-it#command-hooks and get it so Rocket deploys with no config, and maybe Danger needs some because there are more parts